lte/gateway/c/core/oai/tasks/sgw/spgw_state_converter.cpp

```diff
@@ -16,11 +16,11 @@
  */
 
 #include "lte/gateway/c/core/oai/tasks/sgw/spgw_state_converter.hpp"
+#include "lte/gateway/c/core/oai/include/sgw_context_manager.hpp"
```

Comment:
Move this after the `extern "C"` block as per [style guide](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes)